content/cloud/noops-sql-db.md

```diff
@@ -34,69 +36,63 @@ Once you have access to the private beta, please ensure you have [Spin CLI](./cl
-[component]
-name = "c1"
+## Tables and Data in Fermyon Cloud
 
-# c2 can use the default database
-[component]
-name = "c2"
-sqlite_databases = ["default"]
+When you first `spin cloud deploy` your application (with the `sqlite_databases` configuration, as shown above), Fermyon Cloud will create a new & empty database for your Cloud application. This new Cloud database will have a unique name. To obtain the name, type the following command in your local NoOps SQLite application's directory:
 
-# c3 can use default database as well
-[component]
-name = "c3"
-sqlite_databases = ["default"]
+<!-- @selectiveCpy -->
+
+```bash
+$ spin cloud sqlite list
+Databases (1)
+inspirational-pig (default)
 ```
 
-The database will be created when the Spin application is deployed. The data persists Spin application invocations and updates. 
+Then use the `spin cloud sqlite execute` command to create your Cloud database's schema i.e. execute language syntax compatible with [SQLite](https://www.sqlite.org/lang.html) against your Cloud database.
 
 ```bash
-$ spin cloud deploy
-Deploying app ..
-Creating database inspirational-pig (default) for app "quickstart"
+$ spin cloud sqlite execute inspirational-pig "CREATE TABLE IF NOT EXISTS todos (id INTEGER PRIMARY KEY AUTOINCREMENT,description TEXT NOT NULL,due_date DATE,starred BOOLEAN DEFAULT 0,is_completed BOOLEAN DEFAULT 0)"
 ```
 
-You can check to see your database by using the `spin cloud sqlite` command:
+> Note: Cloud database is completely unrelated to the local (optional) database.
 
->> Fermyon Cloud will automatically generate a random name for your database.
+## Deleting the Cloud Database
 
-```bash
-$ spin cloud sqlite list
-inspirational-pig (default) 
-```
+Warning: using the `delete` subcommand is **permanent**. 
 
-Lastly, (warning; using the `delete` subcommand is **permanent**) you can delete your default database with the following command:
+You can delete your default database with the following command:
 
 ```bash
 $ spin cloud sqlite delete inspirational-pig
 The action is irreversible.
-Please type "strong-boson" for confirmation: strong-boson
+Please type "inspirational-pig" for confirmation: inspirational-pig
 Deleting database ...
-Database "strong-boson" deleted
+Database "inspirational-pig" deleted
 ```
 
-> In the Private Beta, the database is inaccessible once the application has been deleted. However, we separate out the the database deletion step because we anticipate the database being accessible after deletion in a future beta. We enforce database deletion to prevent unintentional data sharing between Spin applications. 
+> In the Private Beta, the database is inaccessible once the application has been deleted. However, we separate out the database deletion step because we anticipate the database being accessible after deletion in a future beta. We enforce database deletion to prevent unintentional data sharing between Spin applications. 
```

Comment:
The database is currently accessible after an application is deleted. It is that we currently do not support moving a database from one app to another.

Another point to this is that we do not currently support exporting a database's data before deletion. I think that is a goal too.
```suggestion
> In the Private Beta, we do not have support for sharing database between apps. However, we still separate out the database deletion from app deletion because we anticipate the ability to use databases from multiple apps in the future.
```